src/templates/csr/capi-multiple-paidfor/test.json

```diff
@@ -1,21 +1,21 @@
 {
```

Comment:
Is there a reason why we changed the tests to be empty strings?

Reply:
They're optional fields so don't need to be specified for the test creative